### ISSUE TITLE
New submissions for Thu, 13 Jan 22

### ISSUE THREAD

**Issue by LeeKyungwook** [LeeKyungwook/get-arxiv-noti#198]
### A Feature Extraction based Model for Hate Speech Identification
 - **Authors:** Authors: Salar Mohtaj, Vera Schmitt, Sebastian Möller
 - **Subjects:** Computation and Language (cs.CL); Artificial Intelligence (cs.AI); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2201.04227
 - **Pdf link:** https://arxiv.org/pdf/2201.04227
 - **Abstract**
 The detection of hate speech online has become an important task, as offensive language such as hurtful, obscene and insulting content can harm marginalized people or groups. This paper presents TU Berlin team experiments and results on the task 1A and 1B of the shared task on hate speech and offensive content identification in Indo-European languages 2021. The success of different Natural Language Processing models is evaluated for the respective subtasks throughout the competition. We tested different models based on recurrent neural networks in word and character levels and transfer learning approaches based on Bert on the provided dataset by the competition. Among the tested models that have been used for the experiments, the transfer learning-based models achieved the best results in both subtasks.
### SmartDet: Context-Aware Dynamic Control of Edge Task Offloading for  Mobile Object Detection
 - **Authors:** Authors: Davide Callegaro, Francesco Restuccia, Marco Levorato
 - **Subjects:** Distributed, Parallel, and Cluster Computing (cs.DC); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG); Networking and Internet Architecture (cs.NI)
 - **Arxiv link:** https://arxiv.org/abs/2201.04235
 - **Pdf link:** https://arxiv.org/pdf/2201.04235
 - **Abstract**
 Mobile devices increasingly rely on object detection (OD) through deep neural networks (DNNs) to perform critical tasks. Due to their high complexity, the execution of these DNNs requires excessive time and energy. Low-complexity object tracking (OT) can be used with OD, where the latter is periodically applied to generate "fresh" references for tracking. However, the frames processed with OD incur large delays, which may make the reference outdated and degrade tracking quality. Herein, we propose to use edge computing in this context, and establish parallel OT (at the mobile device) and OD (at the edge server) processes that are resilient to large OD latency. We propose Katch-Up, a novel tracking mechanism that improves the system resilience to excessive OD delay. However, while Katch-Up significantly improves performance, it also increases the computing load of the mobile device. Hence, we design SmartDet, a low-complexity controller based on deep reinforcement learning (DRL) that learns controlling the trade-off between resource utilization and OD performance. SmartDet takes as input context-related information related to the current video content and the current network conditions to optimize frequency and type of OD offloading, as well as Katch-Up utilization. We extensively evaluate SmartDet on a real-world testbed composed of a JetSon Nano as mobile device and a GTX 980 Ti as edge server, connected through a Wi-Fi link. Experimental results show that SmartDet achieves an optimal balance between tracking performance - mean Average Recall (mAR) and resource usage. With respect to a baseline with full Katch-Upusage and maximum channel usage, we still increase mAR by 4% while using 50% less of the channel and 30% power resources associated with Katch-Up. With respect to a fixed strategy using minimal resources, we increase mAR by 20% while using Katch-Up on 1/3 of the frames.
### Incidents1M: a large-scale dataset of images with natural disasters,  damage, and incidents
 - **Authors:** Authors: Ethan Weber, Dim P. Papadopoulos, Agata Lapedriza, Ferda Ofli, Muhammad Imran, Antonio Torralba
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2201.04236
 - **Pdf link:** https://arxiv.org/pdf/2201.04236
 - **Abstract**
 Natural disasters, such as floods, tornadoes, or wildfires, are increasingly pervasive as the Earth undergoes global warming. It is difficult to predict when and where an incident will occur, so timely emergency response is critical to saving the lives of those endangered by destructive events. Fortunately, technology can play a role in these situations. Social media posts can be used as a low-latency data source to understand the progression and aftermath of a disaster, yet parsing this data is tedious without automated methods. Prior work has mostly focused on text-based filtering, yet image and video-based filtering remains largely unexplored. In this work, we present the Incidents1M Dataset, a large-scale multi-label dataset which contains 977,088 images, with 43 incident and 49 place categories. We provide details of the dataset construction, statistics and potential biases; introduce and train a model for incident detection; and perform image-filtering experiments on millions of images on Flickr and Twitter. We also present some applications on incident analysis to encourage and enable future work in computer vision for humanitarian aid. Code, data, and models are available at this http URL
### MDS-Net: A Multi-scale Depth Stratification Based Monocular 3D Object  Detection Algorithm
 - **Authors:** Authors: Zhouzhen Xie, Yuying Song, Jingxuan Wu, Zecheng Li, Chunyi Song, Zhiwei Xu
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2201.04341
 - **Pdf link:** https://arxiv.org/pdf/2201.04341
 - **Abstract**
 Monocular 3D object detection is very challenging in autonomous driving due to the lack of depth information. This paper proposes a one-stage monocular 3D object detection algorithm based on multi-scale depth stratification, which uses the anchor-free method to detect 3D objects in a per-pixel prediction. In the proposed MDS-Net, a novel depth-based stratification structure is developed to improve the network's ability of depth prediction by establishing mathematical models between depth and image size of objects. A new angle loss function is then developed to further improve the accuracy of the angle prediction and increase the convergence speed of training. An optimized soft-NMS is finally applied in the post-processing stage to adjust the confidence of candidate boxes. Experiments on the KITTI benchmark show that the MDS-Net outperforms the existing monocular 3D detection methods in 3D detection and BEV detection tasks while fulfilling real-time requirements.
### Theoretical Limits of Joint Detection and Estimation for Radar Target
 - **Authors:** Authors: Nan Wang, Dazhuan Xu
 - **Subjects:** Information Theory (cs.IT)
 - **Arxiv link:** https://arxiv.org/abs/2201.04342
 - **Pdf link:** https://arxiv.org/pdf/2201.04342
 - **Abstract**
 This paper proposes a joint detection and estimation (JDE) scheme based on mutual information for the radar work, whose goal is to choose the true one between target existent and target absence, and to estimate the unknown distance parameter when the target is existent. Inspired by the thoughts of Shannon information theory, the JDE system model is established in the presence of complex white Gaussian noise. We make several main contributions: (1) the equivalent JDE channel and the posterior probability density function are derived based on the priori statistical characteristic of the noise, target scattering and joint target parameter; (2) the performance of the JDE system is measured by the joint entropy deviation and the joint information that is defined as the mutual information between received signal and the joint target parameter; (3) the sampling a posterior probability and cascaded JDEers are proposed, and their performance is measured by the empirical joint entropy deviation the empirical joint information; (4) the joint theorem is proved that the joint information is the available limit of the overall performance, that is, the joint information is available, and the empirical joint information of any JDEer is no greater than the joint information; (5) the cascaded theorem is proved that the sum of empirical detection information and empirical estimation information can approximate the joint information, i.e., the performance limit of cascaded JDEer is available. Simulation results verify the correctness of the joint and the cascaded theorems, and show that the performance of the sampling a posterior probability JDEer is asymptotically optimal. Moreover, the performance of cascaded JDEer can approximate the system performance of JDE system.
### Detecting Ransomware Execution in a Timely Manner
 - **Authors:** Authors: Anthony Melaragno, William Casey
 - **Subjects:** Cryptography and Security (cs.CR); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2201.04424
 - **Pdf link:** https://arxiv.org/pdf/2201.04424
 - **Abstract**
 Ransomware has been an ongoing issue since the early 1990s. In recent times ransomware has spread from traditional computational resources to cyber-physical systems and industrial controls. We devised a series of experiments in which virtual instances are infected with ransomware. We instrumented the instances and collected resource utilization data across a variety of metrics (CPU, Memory, Disk Utility). We design a change point detection and learning method for identifying ransomware execution. Finally we evaluate and demonstrate its ability to detect ransomware efficiently in a timely manner when trained on a minimal set of samples. Our results represent a step forward for defense, and we conclude with further remarks for the path forward.
### Jamming Detection for IR-UWB Ranging Technology in Autonomous UAV Swarms
 - **Authors:** Authors: Pavlo Mykytyn, Marcin Brzozowski, Zoya Dyka, Peter Langendoerfer
 - **Subjects:** Cryptography and Security (cs.CR)
 - **Arxiv link:** https://arxiv.org/abs/2201.04425
 - **Pdf link:** https://arxiv.org/pdf/2201.04425
 - **Abstract**
 Jamming is a form of the Denial of Service (J-DoS) attack. It is a significant threat that causes malfunction in Unmanned Aerial Vehicle systems, especially when used in hostile environments. The attackers mainly operate in the wireless communication environment by following a few preexisting scenarios. In this paper, we propose an idea for a Jamming detection mechanism. The mechanism utilizes the network parameters available to the system and some additional measures to distinguish between bad transmission quality and Jamming to avoid false positive alarms. After detecting a Jamming attack, appropriate countermeasures or mitigation techniques can be applied to keep the system safe.
### Machine-learning-aided Massive Hybrid Analog and Digital MIMO DOA  Estimation for Future Wireless Networks
 - **Authors:** Authors: Feng Shu, Xichao Zhan, Wenlong Cai, Mengxing Huang, Qijuan Jie, Yifang Li, Baihua Shi, Jiangzhou Wang
 - **Subjects:** Information Theory (cs.IT); Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2201.04452
 - **Pdf link:** https://arxiv.org/pdf/2201.04452
 - **Abstract**
 Due to a high spatial angle resolution and low circuit cost of massive hybrid analog and digital (HAD) multiple-input multiple-output (MIMO), it is viewed as a key technology for future wireless networks. Combining a massive HAD-MIMO with direction of arrinal (DOA) will provide a high-precision even ultra-high-precision DOA measurement performance approaching the fully-digital (FD) MIMO. However, phase ambiguity is a challenge issue for a massive HAD-MIMO DOA estimation. In this paper, we review three aspects: detection, estimation, and Cramer-Rao lower bound (CRLB) with low-resolution ADCs at receiver. First, a multi-layer-neural-network (MLNN) detector is proposed to infer the existence of passive emitters. Then, a two-layer HAD (TLHAD) MIMO structure is proposed to eliminate phase ambiguity using only one-snapshot. Simulation results show that the proposed MLNN detector is much better than both the existing generalized likelihood ratio test (GRLT) and the ratio of maximum eigen-value (Max-EV) to minimum eigen-value (R-MaxEV-MinEV) in terms of detection probability. Additionally, the proposed TLHAD structure can achieve the corresponding CRLB using single snapshot.
### Transmission Scheme, Detection and Power Allocation for Uplink User  Cooperation with NOMA and RSMA
 - **Authors:** Authors: Omid Abbasi, Halim Yanikomeroglu
 - **Subjects:** Information Theory (cs.IT); Networking and Internet Architecture (cs.NI)
 - **Arxiv link:** https://arxiv.org/abs/2201.04572
 - **Pdf link:** https://arxiv.org/pdf/2201.04572
 - **Abstract**
 In this paper, we propose two novel cooperative-non-orthogonal-multiple-access (C-NOMA) and cooperative-rate-splitting-multiple-access (C-RSMA) schemes for uplink user cooperation. At the first mini-slot of these schemes, each user transmits its signal and receives the transmitted signal of the other user in full-duplex mode, and at the second mini-slot, each user relays the other user's message with amplify-and-forward (AF) protocol. At both schemes, to achieve better spectral efficiency, users transmit signals in the non-orthogonal mode in both mini-slots. In C-RSMA, we also apply the rate-splitting method in which the message of each user is divided into two streams. In the proposed detection schemes for C-NOMA and C-RSMA, we apply a combination of maximum-ratio-combining (MRC) and successive-interference-cancellation (SIC). Then, we derive the achievable rates for C-NOMA and C-RSMA, and formulate two optimization problems to maximize the minimum rate of two users by considering the proportional fairness coefficient. We propose two power allocation algorithms based on successive-convex-approximation (SCA) and geometric-programming (GP) to solve these non-convex problems. Next, we derive the asymptotic outage probability of the proposed C-NOMA and C-RSMA schemes, and prove that they achieve diversity order of two. Finally, the above-mentioned performance is confirmed by simulations.
### Sound Dr -- A database of Respiratory Sound and Baseline System for  COVID-19 Detection
 - **Authors:** Authors: Hoang Van Truong, Nguyen Thi Anh Sinh, Nguyen Quoc Cuong, Nguyen Xuan Phong
 - **Subjects:** Sound (cs.SD); Audio and Speech Processing (eess.AS)
 - **Arxiv link:** https://arxiv.org/abs/2201.04581
 - **Pdf link:** https://arxiv.org/pdf/2201.04581
 - **Abstract**
 As the COVID-19 pandemic significantly affect every aspects of human life, it is urgent to provide a data for further researches. We, therefore, introduce a dataset named Sound Dr Database which provides not only quality sounds of coughing and breathing but also metadata for relevant-respiratory illness or diseases. Creating proof-of-concept systems is effective for the detection of abnormalities in the respiratory sounds of patients. These solutions will serve as effective tools to assist physicians in diagnosing respiratory disorders.
### Towards a Catalog of Composite Refactorings
 - **Authors:** Authors: Aline Brito, Andre Hora, Marco Tulio Valente
 - **Subjects:** Software Engineering (cs.SE)
 - **Arxiv link:** https://arxiv.org/abs/2201.04599
 - **Pdf link:** https://arxiv.org/pdf/2201.04599
 - **Abstract**
 Catalogs of refactoring have key importance in software maintenance and evolution, since developers rely on such documents to understand and perform refactoring operations. Furthermore, these catalogs constitute a reference guide for communication between practitioners since they standardize a common refactoring vocabulary. Fowler's seminal book on refactoring describes the most popular catalog of refactorings, which documents single and well-known refactoring operations. However, sometimes refactorings are composite transformations, i.e., a sequence of refactorings performed over a given program element. For example, a sequence of Extract Method operations (a single refactoring) can be performed over the same method, in one or in multiple commits, to simplify its implementation, therefore, leading to a Method Decomposition operation (a composite refactoring). In this paper, we propose and document a catalog with six composite refactorings. We also extend a refactoring detection tool to identify such composite refactoring in commits histories. We rely on our tool extension to mine composite refactorings in ten well-known open source projects and characterize the detected composite refactorings, under dimensions such as size and location. Finally, we qualitatively study the descriptions of commits including composite refactorings to shed preliminary light on the motivations behind such operations. We conclude by addressing the applications and implications of the proposed catalog.
### Sparsely Annotated Object Detection: A Region-based Semi-supervised  Approach
 - **Authors:** Authors: Sai Saketh Rambhatla, Saksham Suri, Rama Chellappa, Abhinav Shrivastava
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2201.04620
 - **Pdf link:** https://arxiv.org/pdf/2201.04620
 - **Abstract**
 Research shows a noticeable drop in performance of object detectors when the training data has missing annotations, i.e. sparsely annotated data. Contemporary methods focus on proxies for missing ground-truth annotations either in the form of pseudo-labels or by re-weighing gradients for unlabeled boxes during training. In this work, we revisit the formulation of sparsely annotated object detection. We observe that sparsely annotated object detection can be considered a semi-supervised object detection problem at a region level. Building on this insight, we propose a region-based semi-supervised algorithm, that automatically identifies regions containing unlabeled foreground objects. Our algorithm then processes the labeled and un-labeled foreground regions differently, a common practice in semi-supervised methods. To evaluate the effectiveness of the proposed approach, we conduct exhaustive experiments on five splits commonly used by sparsely annotated approaches on the PASCAL-VOC and COCO datasets and achieve state-of-the-art performance. In addition to this, we show that our approach achieves competitive performance on standard semi-supervised setups demonstrating the strength and broad applicability of our approach.
### Beyond the Visible: A Survey on Cross-spectral Face Recognition
 - **Authors:** Authors: David Anghelone, Cunjian Chen, Arun Ross, Antitza Dantcheva
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2201.04435
 - **Pdf link:** https://arxiv.org/pdf/2201.04435
 - **Abstract**
 Cross-spectral face recognition (CFR) is aimed at recognizing individuals, where compared face images stem from different sensing modalities, for example infrared vs. visible. While CFR is inherently more challenging than classical face recognition due to significant variation in facial appearance associated to a modality gap, it is superior in scenarios with limited or challenging illumination, as well as in the presence of presentation attacks. Recent advances in artificial intelligence related to convolutional neural networks (CNNs) have brought to the fore a significant performance improvement in CFR. Motivated by this, the contributions of this survey are three-fold. We provide an overview of CFR, targeted to compare face images captured in different spectra, by firstly formalizing CFR and then presenting concrete related applications. Secondly, we explore suitable spectral bands for recognition and discuss recent CFR-methods, placing emphasis on deep neural networks. In particular we revisit techniques that have been proposed to extract and compare heterogeneous features, as well as datasets. We enumerate strengths and limitations of different spectra and associated algorithms. Finally, we discuss research challenges and future lines of research.
## Keyword: augmentation
There is no result
